### PR TITLE
area type doc

### PR DESCRIPTION
**Pull request by djamnazi**
form type doc: AreaType

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 en/_build
+.idea/
```
Comment (alavieille) [inline .gitignore]: remove this line, you can ignore .idea file in your local gitignore

File: en/developer_guide/form/ApiClientType.rst [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+|                                   |                                                  |
++-----------------------------------+--------------------------------------------------+
+| **Parent type**                   | FormType                                         |
+|                                   |                                                  |
++-----------------------------------+--------------------------------------------------+
+| **Options**                       |  * inherited options FormType                    |
+|                                   |                                                  |
+|                                   |                                                  |
++-----------------------------------+--------------------------------------------------+
+| **Event subscriber**              |                                                  |
```
Comment (alavieille) [inline en/developer_guide/form/ApiClientType.rst]: if there isn't Event subscriber remove this section

File: en/developer_guide/form/ApiClientType.rst [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+|                                   |                                                  |
++-----------------------------------+--------------------------------------------------+
+| **View transformer**              |                                                  |
```
Comment (alavieille) [inline en/developer_guide/form/ApiClientType.rst]: same

File: en/developer_guide/form/ApiClientType.rst [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+ApiClientType
+=============
+
+
+The ApiClientType form is used to create a workflow role in Back Office.
+
++-----------------------------------+--------------------------------------------------+
+| **Bundle**                        | OpenOrchestraBackofficeBundle                    |
++-----------------------------------+--------------------------------------------------+
+| **Name**                          | oo_api_client                                          |
```
Comment (alavieille) [inline en/developer_guide/form/ApiClientType.rst]: align |

File: en/developer_guide/form/ApiClientType.rst [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
++-----------------------------------+--------------------------------------------------+
+| **Class**                         | ApiClientType                                    |
+|                                   |                                                  |
++-----------------------------------+--------------------------------------------------+
+| **Parent type**                   | FormType                                         |
+|                                   |                                                  |
++-----------------------------------+--------------------------------------------------+
+| **Options**                       |  * inherited options FormType                    |
+|                                   |                                                  |
+|                                   |                                                  |
```
Comment (alavieille) [inline en/developer_guide/form/ApiClientType.rst]: remove this useless line

File: en/developer_guide/form/ApiClientType.rst [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
++-----------------------------------+--------------------------------------------------+
+
+
+Inherited Options:
+==================
+
+ Options inherit from the `FormType <http://symfony.com/doc/current/reference/forms/types/form.html>`_
+
+
+Options:
```
Comment (alavieille) [inline en/developer_guide/form/ApiClientType.rst]: remove this section if there isn't options

File: en/developer_guide/form/AreaType.rst [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
++-----------------------------------+-----------------------------------+
+
+
+Inherited Options:
+==================
+
+ Options inherit from the `FormType <http://symfony.com/doc/current/reference/forms/types/form.html>`_
+
+
+Options:
```
Comment (alavieille) [inline en/developer_guide/form/AreaType.rst]: remove this part

**Comment by djamnazi** [comment]
@alavieille PR updated

**Comment by alavieille** [comment]
i think your are forget fom which are in component folder

**Comment by djamnazi** [comment]
@alavieille Pr updated

**Pull request closed without merging**